include/cute/arch/util.hpp

```diff
@@ -34,7 +34,7 @@
 
 #include <cute/numeric/integer_sequence.hpp>
 
-#if defined(__clang__) && defined(__CUDA__)
+#if defined(__clang__) && defined(__CUDA__) && defined(__CUDA_ARCH__)
```

Comment:
Most likely this qualification on `defined(__CUDA_ARCH__)` should go down on line # 64. The SUPPORTS or SUPPORTED flags define whether the compiler has the support in general, and then the ACTIVATED definitions qualify the supports flag on whether we're doing device compilation (when applicable).